io.github.mak448a.QTCord.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.mak448a.QTCord
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
```

Comment:
I'm pretty sure this should be 6.6, since you are using pyside 6.6.1 which is Qt 6.6.1